tests/helics/test_files/multi_input_config.json

```diff
@@ -0,0 +1,33 @@
+//this should be a valid json file (except comments are not recognized in standard JSON)
+{
+  "name": "valueFed_multi", // the name of the federate
+  //possible flags
+  "coretype": "test", //the type of the core "test","zmq","udp","ipc","tcp","mpi"
+  "corename": "multi_core1", //this matters most for ipc and inproc cores, can be empty
+  "coreinit": "--autobroker", // the initialization string for the core in the form of a command line arguments
+
+  //Publications used in the federate
+  "publications": [
+    {
+      "key": "pub1", // the name of the publication
+      "type": "double", // the type associated with a publication (optional)
+      "global": true, //set to true to make the key global (default is false in which case the publication is prepended with the federate name)
+      "info": "this is an information string for use by the application"
+    },
+    {
+      "key": "pub2", // the name of the publication
+      "type": "double", // the type associated with a publication (optional)
+      "global": true
+    }
+  ],
+
+  "inputs": [
+    {
+      "key": "ipt2",
+      "type": "double",
+      "connections": 2,
+      "multi_input_handling_method": "average",
+      "targets": ["pub1", "pub2"]
```

Comment:
To change priorities with an input file, would a user just change the order of items in the targets list?

Reply:
Generally yes,  Maybe should make a comment in the docs to that effect.  But that is not technically guaranteed, but the only situation in which it wouldn't occur is if there was intermediate brokers and the later publications were on the intermediate ones and the earlier were not.    I will fix that in HELICS 3.0 to guarantee an ordering but that requires a small change in the message which would break some compatibility